javascript/packages/orchestrator/src/network.ts

```diff
@@ -95,16 +95,23 @@ export class Network {
   wasRunning: boolean;
   tmpDir: string;
   backchannelUri: string = "";
+  chainId?: string;
   chainSpecFullPath?: string;
   tracing_collator_url?: string;
   networkStartTime?: number;
```

Comment:
```suggestion
  backchannelUri: string = "";
  chainId?: string;
  chainSpecFullPath?: string;
  tracing_collator_url?: string;
```

Can do the same with ```namespace``` and ```client``` above

Reply:
same as above; also your proposal is removing some variables - why?
```suggestion
  tmpDir: string;
  backchannelUri = "";
  chainId?: string;
  chainSpecFullPath?: string;
  tracing_collator_url?: string;
  networkStartTime?: number;
```

Reply:
Yes, because we are using constructor property definition below, we can remove them from here